src/Compilers/CSharp/Portable/Binder/Binder_Invocation.cs

```diff
@@ -1411,7 +1411,14 @@ BoundExpression bindDefaultArgument(SyntaxNode syntax, ParameterSymbol parameter
                     {
                         GenerateImplicitConversionError(diagnostics, syntax, conversion, defaultValue, parameterType);
                     }
-                    defaultValue = CreateConversion(defaultValue, conversion, parameterType, diagnostics);
+                    defaultValue = CreateConversion(
+                        defaultValue.Syntax,
+                        defaultValue,
+                        conversion,
+                        isCast: conversion.IsExplicit,
+                        new ConversionGroup(conversion, parameter.TypeWithAnnotations),
```

Comment:
From our offline discussion: consider avoiding this allocation when `isCast` is `false`

Reply:
Done